proxy/src/auth/backend/web.rs

```diff
@@ -41,15 +44,16 @@ impl ReportableError for WebAuthError {
     }
 }
 
-fn hello_message(redirect_uri: &reqwest::Url, session_id: &str) -> String {
+fn hello_message(redirect_uri: &reqwest::Url, session_id: &str, timeout: Duration) -> String {
     format!(
         concat![
             "Welcome to Neon!\n",
-            "Authenticate by visiting:\n",
+            "Authenticate by visiting within {timeout}:\n",
             "    {redirect_uri}{session_id}\n\n",
```

Comment:
I suppose that's not quite correct. You have to also login and choose the database within this time. Maybe something along the lines of

Authenticate by visiting
<link>
And choose a database within <time> to connect

Reply:
Yeah. Not sure about that. Perhaps instead we could make it clear at the timeout with an appropriate error message that includes the timeout duration?
"you were disconnected due to inactivity after 2 minutes" or something

Reply:
What is the downside of announcing? Attackers? I guess they can figure it out easily in any case...

Reply:
It just bloats the output